compoments/report-repair/repair-availability.js

```diff
@@ -36,12 +36,19 @@ const RepairAvailability = ({handleChange, values, nextAvailability}) => {
         availability[dateString] = [timeString]
     })
   }
+  const convertDateToDisplayDate = (date) => {
```

Comment:
this is redundant, simply change `to` on [line 33](https://github.com/City-of-Lincoln-Council/housing-repairs-online-frontend/pull/33/files#diff-50a0eb2e4b5dfda7f90ef25469bbb24709e3f9dbdd5dfb65ad42694f3e9c57fcR33) to be `between`